lib/elasticband/filter/range.rb

```diff
@@ -0,0 +1,29 @@
+module Elasticband
+  class Filter
+    class Range < Filter
+      attr_accessor :field, :ranges
+
+      RANGES = %i(gt gteq lt lteq time_zone)
```

Comment:
I do not understand as well, I'm just allowing it because the elasticsearch has this option, but I can remove it since we are not going to use it. ;) What do you think?